roles/core/nic_nmcli/readme.rst

```diff
@@ -175,14 +175,17 @@ time set their configuration.
 However, in some cases, users might need to force some updates (for example if 
 you wish to set routes on the main interface, etc).
 
-To achieve that, two variables are at disposal:
+To achieve that, few variables are at disposal:
```

Comment:
Perhaps reword toL

_To achieve this, a few variables are at your disposal:_

Reply:
Thanks for correction, done :-)